lms/static/js/search/course/views/search_form.js

```diff
@@ -6,7 +6,7 @@ define([
     'use strict';
 
     return SearchForm.extend({
-        el: '#courseware-search-bar'
+        el: '#courseware-search-bar-HACK'
```

Comment:
This prevents event binding to search input in courseware because we redirect to search page when user hits enter.